wrappers/opencv/grabcuts/rs-grabcuts.cpp

```diff
@@ -61,7 +62,7 @@ int main(int argc, char * argv[]) try
 
         // Generate "near" mask image:
         auto near = frame_to_mat(bw_depth);
-        cvtColor(near, near, CV_BGR2GRAY);
+        cvtColor(near, near, COLOR_BGR2GRAY);
```

Comment:
there is another CV_BGR2GRAY that needs to be changed to COLOR_BGR2GRAY a few lines below (l.71)